geopandas/tests/test_array.py

```diff
@@ -480,9 +474,7 @@ def test_unary_predicates(attr):
         ]
     elif attr == "is_ring":
         expected = [
-            getattr(t.exterior, attr)
-            if t is not None and t.exterior is not None
-            else na_value
+            getattr(t, attr) if t is not None and t.exterior is not None else na_value
```

Comment:
I don't fully follow this change (`t` here is a shapely geometry, so that shouldn't change?)